hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2143,57 +2187,65 @@
   </property>
   <property>
     <name>ozone.recon.http.auth.kerberos.keytab</name>
-    <value/>
-    <tag>RECON, SECURITY</tag>
+    <value>/path/to/HTTP.keytab</value>
+    <tag>RECON, SECURITY, KERBEROS</tag>
     <description>
       The keytab file for HTTP Kerberos authentication in Recon.
     </description>
   </property>
   <property>
     <name>ozone.recon.http.auth.kerberos.principal</name>
-    <value/>
-    <tag>RECON</tag>
+    <value>HTTP/_HOST@REALM</value>
+    <tag>RECON, KERBEROS</tag>
```

Comment:
Add tag SECURITY  in line with other similar configs.

Reply:
@symious any thoughts about this?

Reply:
@dineshchitlangia Thanks for the review, SECURITY tag added.